data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -173,8 +173,7 @@
     # won't work with mocked strings.  should use base class
     # setup_query_execution function to load dependencies before query execution
```

Comment:
this comment should now be removed.  It was specific to the old style smoking rule that you are updating.

Reply:
Sorry, I thought I had. It will be fixed with the next commit.